tox.ini

```diff
@@ -19,7 +19,7 @@ ignore = E501,E265,E241,E704
 
 [testenv:docs]
 deps =
-  --requirement={toxinidir}/dev/req_dev.txt
+  --requirement={toxinidir}/dev/req_docs.txt
```

Comment:
I'm surprised this works without req_dev.txt too. Can you confirm it actually does work as written?

Reply:
can confirm, make docs passes provided services code is cleaned up